fe2-android/app/src/main/res/values/strings.xml

```diff
@@ -263,6 +268,7 @@
   <string name="error_storage_wallet">An error occurred during the storage of the wallet seed</string>
   <string name="qr_code_not_pop_token">Error, this QR code is not a pop token or the key format is invalid</string>
   <string name="qr_code_not_main_pk">Error, this QR code is not a main public key or the key format is invalid</string>
+  <string name="error_event_observed">Error, updating events unsuccessful</string>
```

Comment:
this is missing a verb: 
- Error, update events _was_ unsuccessful  ?
- Error, could not update events ?

Also, can the user do anything about this or is it one of those "this should never ever happen" messages ?

Reply:
I'll go with option 2 :) 

This should be very rare, possibly never happening. Nevertheless it means that the observation of events raised an error. Provided nothing bad happened in the repo, coming back to that screen should fix it.